system/lib/standalone/standalone.c

```diff
@@ -176,11 +176,13 @@ imported__wasi_fd_write(__wasi_fd_t fd,
                         __wasi_size_t* nwritten);
 
 static void wasi_write(__wasi_fd_t fd, char* buffer) {
-  struct __wasi_ciovec_t iov;
-  iov.buf = (uint8_t*)buffer;
-  iov.buf_len = strlen(buffer) + 1;
+  struct __wasi_ciovec_t iovs[2];
+  iovs[0].buf = (uint8_t*)buffer;
+  iovs[0].buf_len = strlen(buffer);
+  iovs[1].buf = (uint8_t*)"\n";
+  iovs[1].buf_len = 1;
```

Comment:
Can you rename this method to `wasi_writeln` perhaps?  Or maybe `wasi_puts`?  took me a while to figure out this wasn't being called for every user-level write call.

Reply:
Good point, thanks. Renaming.